src/components/simple_nav.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from "react"
+import styled from "styled-components"
+import colors from "../assets/colors"
+import "../assets/fonts"
+import * as fonts from "../assets/fonts"
+import Icon from "./icon"
+
+const Nav = styled.div`
+  border-bottom: 1px solid ${ colors.grayRegular };
+`
+
+const NavIcon = styled.div`
+  border-right: 1px solid ${ colors.grayRegular };
+  display: inline-block;
+  font-size: 32px;
+  padding: 10px 5px 5px 5px;
+  margin-right: 10px;
+`
+
+const SimpleNav: React.SFC<any> = props => (
```

Comment:
No, it doesn't automatically cast to the correct props type. we should use `React.SFC<HTMLDivElement>` in this case @xtina-starr